builder/bundleset.go

```diff
@@ -221,6 +221,49 @@ func validateBundle(b *bundle) error {
 	return nil
 }
 
+// parsePundleFile parses a file at filename containing a flat list of package
+// names to be converted to one-package bundles (pundles). Returns a list of
+// pointers to bundle objects created from this list.
+func parsePackageBundleFile(filename string) (map[string]bool, error) {
+	packages := make(map[string]bool)
+	contents, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+
+	pundleList := strings.Split(string(contents), "\n")
```

Comment:
Optional: If you wanted, you could use `helpers.ReadFileAndSplit()` to get these same 5 lines.

..but you already have them written.... so..

Reply:
*sigh* fine...

Reply:
Actually, never mind. bundleset doesn't currently import helpers and I don't see the point in doing it for a simple file read/line splitting function.

Reply:
Good answer.